.travis.yml

```diff
@@ -27,7 +30,9 @@ env:
 #- make dep-install
 #- make dep-update
 
-script: make test
+script: 
+  - test -z $(gofmt -s -l $GO_FILES) # Gofmt Linter
```

Comment:
you might have to wrap the -z option value in quotes - this might be what travis failed on. So try this:

`- test -z "$(gofmt -s -l $GO_FILES)" # Gofmt Linter`